src/main/java/model/Lottos.java

```diff
@@ -0,0 +1,44 @@
+package model;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.counting;
+import static java.util.stream.Collectors.groupingBy;
+
+public class Lottos {
+
+    private final List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = new ArrayList<>(lottos);
+    }
+
+    public List<Lotto> getLottos() {
+        return new ArrayList<>(lottos);
+    }
+
+    public BigDecimal calculateYield(WinningLotto winningLotto) {
+        Map<Rank, Long> statistics = getStatistics(winningLotto);
+        BigDecimal benefit = calculateBenefit(statistics);
+        BigDecimal payment = calculatePayment();
+        return benefit.divide(payment);
+    }
+
+    private BigDecimal calculateBenefit(Map<Rank, Long> statistics) {
+        return new BigDecimal(statistics.entrySet().stream()
+                .mapToLong(entry -> entry.getKey().getWinningMoney() * entry.getValue())
+                .sum());
+    }
+
+    private BigDecimal calculatePayment() {
+        return new BigDecimal(lottos.size() * Lotto.PRICE);
+    }
+
+    public Map<Rank, Long> getStatistics(WinningLotto winningLotto) {
+        return lottos.stream()
+                .collect(groupingBy((lotto -> winningLotto.match(lotto)), counting()));
```

Comment:
```suggestion
                .collect(groupingBy(lotto -> winningLotto.match(lotto), counting()));
```
불필요한 괄호는 제거하는게 좋을 것 같습니다. (이 부분은 괄호가 없다고해서 가독성이 떨어지지는 않는 것 같습니다)

Reply:
꼼꼼하게 봐주셔서 감사합니다. 놓쳤던 부분인 것 같습니다.